src/hooks/useRewardsEarned.ts

```diff
@@ -0,0 +1,39 @@
+import { mIOToken } from '@ar.io/sdk/web';
+import { useEffect, useState } from 'react';
+import useEpochs from './useEpochs';
+
+export type RewardsEarned = {
+  previousEpoch: number;
+  totalForPastAvailableEpochs: number;
+};
+
+const useRewardsEarned = (walletAddress?: string) => {
+  const [rewardsEarned, setRewardsEarned] = useState<RewardsEarned>();
```

Comment:
seems logical to maintain this as 0's for the default state no?
```
  previousEpoch: 0;
  totalForPastAvailableEpochs: 0;
```

Reply:
Intentionally using undefined as default to denote loading state, which needs to be differentiated from if a user legitimately did not earn any rewards.